src/EditorFeatures/TestUtilities/EditAndContinue/TestActiveStatementSpanTrackerFactory.cs

```diff
@@ -12,7 +12,6 @@ namespace Microsoft.CodeAnalysis.EditAndContinue.UnitTests
 {
     [ExportWorkspaceService(typeof(IActiveStatementSpanTrackerFactory), ServiceLayer.Host)]
     [Shared]
-    [PartNotDiscoverable]
```

Comment:
❗ Direct exports in test assemblies need this attribute. It provides assurance that they only appear in a composition by explicit request.

Reply:
I had to remove the attribute for MEF2 to find this part even when specified explicitly as a type. I'll revisit.

Reply:
We can also add checks to TestComposition to only accept product assemblies.